js/util.js

```diff
@@ -90,3 +90,33 @@ util.guessLineEndings = function(text) {
 
   return (hasCRLF ? '\r\n' : '\n');
 };
+
+/**
+ * @param {?string} opt_content Optional content.
+ * Creates a unified session that can be read from any supported editor
```

Comment:
nit: description before param here and below

Reply:
Isn't having the description below the param the closure standard?

Reply:
the Google javascript style guide does it above https://google.github.io/styleguide/jsguide.html#jsdoc

Reply:
hm the rest of the codebase does it below? weird, if it's not a huge issue might leave it as is since i just matched the codebase and it doesn't seem to effect the compiler at all.

Reply:
ack. nb the codebase is inconsistent in how it does thism, even within files e.g. tabs.js